src/ensembl/src/shared/types/thoas/transcript.ts

```diff
@@ -33,4 +33,5 @@ export type FullTranscript = {
   product_generating_contexts: FullProductGeneratingContext[];
   external_references: ExternalReference[];
   metadata: TranscriptMetadata;
+  name?: string;
```

Comment:
why? we aren't using transcript names anywhere

Reply:
I added there since the type is `FullTranscript`. I get your point though. `name` is used only in `ensObjectHelpers.ts`, so I'll extend `FullTranscript` with a new type specifically for it.

Reply:
✅